src/components/AppFilter/useAppFilter.test.js

```
@@ -89,38 +86,19 @@ describe('useAppFilter', () => {
     await act(async () => {
       result.current.setIsOpen(true);
     });
-    expect(axiosGetSpy).toHaveBeenCalledTimes(8);
-    for (let index = 0; index < 7; index++) {
-      expect(axiosGetSpy.mock.calls[index]).toEqual([`/api/chrome-service/v1/static/stable/stage/navigation/${requiredBundles[index]}-navigation.json?ts=666`]);
-    }
-    axiosGetSpy.mockReset();
-    dateSpy.mockRestore();
+    expect(result.current.data['openshift'].links).toEqual([
+      expect.objectContaining({ title: 'Clusters', href: '/openshift/clusters' }),
+    ]);
```

ESLint

[error] Replace `⏎······expect.objectContaining({·title:·'Clusters',·href:·'/openshift/clusters'·}),⏎····` with `expect.objectContaining({·title:·'Clusters',·href:·'/openshift/clusters'·})`

(prettier/prettier)